src/Backend/Synapse/SynapseException.php

```diff
@@ -23,7 +59,7 @@ public static function covertException(DBALException $e): Throwable
 
             // strip query from message, there are things like SAS tokens and internal table names
             $message = (string) strstr($e->getMessage(), self::BULK_LOAD_EXCEPTION_BEGINNING);
-            return new Exception(
+            return new self(
```

Comment:
Pokud je tu self a ne static, tak neměla by ta classa být final? (jinak pokud to někdo podědí, tak mu to bude vyhazovat synapse exception místo té jeho.

Reply:
static tam nemože byt to by ti stan dal přes prsty dal sem tam final asi není potřeba to dědit.